### PR TITLE
bump `_version.py` to 0.7.0.dev0

### PR DESCRIPTION
**Pull request by eitanturok**
# What does this PR do?

# What issue(s) does this change relate to?

# Before submitting
- [ ] Have you read the [contributor guidelines](https://github.com/databricks/megablocks/blob/dev/CONTRIBUTING.md)?
- [ ] Is this change a documentation change or typo fix? If so, skip the rest of this checklist.
- [ ] Was this change discussed/approved in a GitHub issue first? It is much more likely to be merged if so.
- [ ] Did you update any related docs and document your change?
- [ ] Did you update any related tests and add any new tests related to your change? (see [testing](https://github.com/databricks/megablocks/blob/dev/CONTRIBUTING.md#running-tests))
- [ ] Did you run the tests locally to make sure they pass?
- [ ] Did you run `pre-commit` on your change? (see the `pre-commit` section of [prerequisites](https://github.com/databricks/megablocks/blob/dev/CONTRIBUTING.md#prerequisites))

### CODE REVIEW

File: megablocks/_version.py [hunk 1]
```diff
@@ -3,4 +3,4 @@
 
 """The MegaBlocks Version."""
 
-__version__ = '0.5.1'
+__version__ = '0.6.1.dev0'
```
Comment (snarayan21) [inline megablocks/_version.py:6]:
```suggestion
__version__ = '0.6.0.dev0'
```
should be this

**Review by snarayan21 — COMMENTED** [review]
0.6.0 not 0.6.1 haha

**Review by snarayan21 — APPROVED** [review]
lgtm!

**Review by dakinggg — APPROVED** [review]

**Review by snarayan21 — CHANGES_REQUESTED** [review]
blocking merge until after i patch release

**Pull request merged**